TrkHitReco/fcl/prolog_trigger.fcl

```diff
@@ -65,29 +60,29 @@ TTSmakePH : {
 # combine panel hits in a station
 TTmakeSTH : {
     @table::makeSTH
-    TestFlag : false
+    TestFlag           : false
     ComboHitCollection : "TTSmakePH"
 }
 TTflagBkgHits : {
-    module_type : FlagBkgHits
-    ComboHitCollection : "TTmakePH"
-    StrawHitCollection : "TTmakeSH"
-    BkgMVA : { MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml" }
-    MVANames : [ "HitRho", "HitRhoSpread", "ClusterRho", "ZMin", "ZMax", "ZGap", "NPlanes", "PlaneFraction","NHits" ] 
-    TLTClusterer : { @table::TLTClusterer 
-	TestFlag : false}
-    TNTClusterer : { @table::TNTClusterer
-	TestFlag : false}
-    # use TNTClusterer
-    Clusterer : 2
-    FilterOutput : true
-    FlagComboHits : true
-    FlagStrawHits : false 
-    ComboHitCollection : "TTmakePH"
-    BkgMVA : {
-	MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml"
-    }
+
+  module_type          : FlagBkgHits
+  ComboHitCollection   : "TTmakePH"
+  StrawHitCollection   : "TTmakeSH"
+  ClusterAlgorithm     : 1
+  TNTClustering        : { @table::TNTClusterer }
+  MinActiveHits        : 3
+  MinNPlanes           : 2
+  #MaxIsolated          : 0  
+  ClusterPositionError : 10
+  BkgMVA               : { MVAWeights : "TrkHitReco/data/BkgMVAPanel.weights.xml" }
```

Comment:
I was thinking that this file comes out from the separation CE from "background", BUT in principle, we may want to use the same module with different configuration and weights for other kinds of track reconstructions, like for instance the protons. So my question is: would it make sense to name it differently or create subdirectories to distinguish the different weight files? @brownd1978 can you please comment?